src/model.h

```diff
@@ -227,6 +208,7 @@ struct RapidsModel : rapids::Model<RapidsSharedState> {
       }
     }
     cpu_model = ForestModel<rapids::HostMemory>(tl_model);
+    cpu_treeshap_model = TreeShapModel<rapids::HostMemory>(tl_model);
```

Comment:
Similar to above comments, `IS_TREESHAP_BUILD` should either be removed or respected, and `cpu_treeshap_model` should be a `std::optional`.

Reply:
Removed.